string obj_class
float64 x
float64 y
float64 z
string id
string color
DBOperation action
---
MetaResponse res